### PR TITLE
chore(concrete_core_wasm): add action to releas concrete-core-wasm

### PR DESCRIPTION
**Pull request by aPere3**
### Description

Adds a github action to build and upload the `concrete-core-wasm` package.

### Checklist 

(Use '[x]' to check the checkboxes, or submit the PR and then click the checkboxes)

* ~[ ] Tests for the changes have been added (for bug fixes / features)~
* ~[ ] Docs have been added / updated (for bug fixes / features)~
* ~[ ] The PR description links to the related issue (to link an issue, use '#XXX'.)~
* ~[ ] The tests on AWS have been launched and are successful (apply the `aws_test` to the PR to launch the tests on AWS)~
* ~[ ] The draft release description has been updated~
* ~[ ] Check for breaking changes and add them to commit message following the conventional commit [specification][conventional-breaking]~

[conventional-breaking]: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

### CODE REVIEW

File: .github/workflows/wasm_github_release.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      run:
+        shell: bash
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build concrete-core-wasm with wasm-pack
+        run: |
+          
+          # Install wasm pack
+          curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
```
Comment (IceTDrinker) [inline .github/workflows/wasm_github_release.yml:27]: is there a sha256 for this script to validate it ?

**Review by IceTDrinker — COMMENTED** [review]
Looks good just a question when downloading a raw shell script that we maybe could validate with a hash

File: .github/workflows/wasm_github_release.yml (hunk 1 above)
Comment (aPere3) [reply .github/workflows/wasm_github_release.yml:27]: I could not find one, but apparently, I can also install with `cargo install wasm-pack` . That would probably be better ...

Comment (IceTDrinker) [reply .github/workflows/wasm_github_release.yml:27]: if it works with cargo it would be great 👍

Comment (aPere3) [reply .github/workflows/wasm_github_release.yml]: It does !

**Review by IceTDrinker — APPROVED** [review]
Thanks a lot!

**Pull request merged**